fallbackinput.go

```diff
@@ -23,15 +22,7 @@ func (s *State) PasswordPrompt(p string) (string, error) {
 	return "", errors.New("liner: function not supported in this terminal")
 }
 
-// NewLiner initializes a new *State
-//
-// Note that this operating system uses a fallback mode without line
-// editing. Patches welcome.
-func NewLiner() *State {
-	var s State
-	s.setWriter(os.Stdout)
-	s.setReader(os.Stdin)
```

Comment:
Style nit: "Fix build for..." type commits should be squashed in before sending a merge request. Leaving them separate is both more difficult to review and causes unnecessary "git bisect" errors in the future.